### DISCUSSION TITLE
Please suggest me your favourite song at any genre!

### DISCUSSION THREAD

**Discussion by thapakorn604** [NAIST-SE/AttractingContributors#1115] [General]
The project runs at: https://github.com/thapakorn604/AttractingContributors-songs

I would like to know your favourite songs in any genre! Please suggest me some so I could have a list to check them out!

**Comment by Ninjia315** [comment]
I have submitted a pull request. if possible, please help me my project as well! thank you!
https://github.com/Ninjia315/AttractingContributors-Favorite_Japanese_songs

**Reply by thapakorn604** [reply]
Sorry, but I don't see your pull request, anyways, I will create my request to yours soon

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite one.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
